lib/db/mysql.js

```diff
@@ -288,12 +288,35 @@ module.exports = function (log, error) {
     )
   }
 
+  // Enum capability name -> capability id (index)
+  const CAPABILITIES = ['pushbox']
```

Comment:
Should we also define a `const CAPABILITY_IDS` at this point, containing the inverse mapping from string to integer? It would save us having to rely on `indexOf` in the rest of the code, and could be generated with `CAPABILITIES.reduce` so no need to actually define anything really.